### DISCUSSION TITLE
Bubble mod should be default 1.0x multiplier

### DISCUSSION THREAD

**Discussion by spongebob84** [ppy/osu#23783] [Ideas]
By default, the bubble mod should be enabled and apply the 1.0x score multiplier. Turning off the mod would be considered a downgrade in multiplier, meaning what is considered no-mod nowadays would be down to .7x with a no-bubbles mod. I played the game with bubble and it just makes sense.

**Comment by monochrome22** [comment]
To be clear, you are suggesting that the Bubbles mod be enabled for all players by default when they start the game? And disabling it would give them a negative score multiplier?

**Reply by spongebob84** [reply]
Yes.

**Reply by monochrome22** [reply]
This will definitely not happen.

If you want Bubbles mod to be more rewarding, then it would make more sense to have a discussion about whether or not it should have a positive multiplier.

**Reply by spongebob84** [reply]
But it shouldn't need to have a positive multiplier. It should be the default. Playing with bubble is how the game should be made to be played. Playing without it feels like cheating.

**Reply by Rekunan** [reply]
> Playing with bubble is how the game should be made to be played.

What's your reasoning for this.

**Comment by bdach** [comment]
glad you like the mod, but there's not a chance of that happening
